pkgs/development/python-modules/matrix-client/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonPackages
+}:
+
+buildPythonPackage rec {
+  pname = "matrix-client";
+  version = "0.0.6";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "15kx5px26hwr0sxpyjk4w61fjnabg1b57hwys1nyarc0jx4qjhiq";
+  };
+
+  buildInputs = with pythonPackages; [ requests ];
+
+  # Tests need extra requirements, disabling for now.
+  doCheck = false;
```

Comment:
Tried to get the tests up and running:

For python2 it worked (but `setup.py test` doesn't seem to do anything. `pytest` does however, so I added a `checkPhase` (possible the setup check is something else?))

Then I tried python3 (which should be supported) but that fails on a simple build (also without doCheck) so consider this PR on halt until I figure out the python3 thing. (I'll \@mention you when I'm done)

Snippet from the failed build in case you have any immediate hunches:

    Processing ./matrix_client-0.0.6-py3-none-any.whl
    Collecting requests (from matrix-client==0.0.6)
      Could not find a version that satisfies the requirement requests (from matrix-client==0.0.6) (from versions: )
    No matching distribution found for requests (from matrix-client==0.0.6)